### ISSUE TITLE
Add "-no-color" option

### ISSUE THREAD

**Issue by sthristov** [tfutils/tfenv#121]
Hello,

First of all, let me just say that I like tfenv and thank you for taking the time to create it.

I'm using tfenv in AWS CodeBuild and it would be nice to have the same "-no-color" option as Terraform has, as now the command output looks like this:

```
·[0;32m[INFO] Installing Terraform v0.12.3·[0;39m 
·[0;32m[INFO] Downloading release tarball from https://releases.hashicorp.com/terraform/0.12.3/terraform_0.12.3_linux_amd64.zip·[0;39m 
 
                                                                           0.6% 
###################################                                       48.9% 
######################################################################## 100.0% 
·[0;32m[INFO] Downloading SHA hash file from https://releases.hashicorp.com/terraform/0.12.3/terraform_0.12.3_SHA256SUMS·[0;39m 
tfenv: tfenv-install: ·[0;33m[WARN] No keybase install found, skipping OpenPGP signature verification·[0;39m 
Archive:  tfenv_download.GiiMAo/terraform_0.12.3_linux_amd64.zip 
  inflating: /************/.tfenv/versions/0.12.3/terraform   
·[0;32m[INFO] Installation of terraform v0.12.3 successful·[0;39m 
·[0;32m[INFO] Switching to v0.12.3·[0;39m 
·[0;32m[INFO] Switching completed·[0;39m
```

Thank you,
Stan

**Comment by Zordrak** [comment]
It's a good ask, and I will see if I can find time for it. It may be that I incorporate this into an intended rearchitecting of the logging mechanism.

**Comment by Zordrak** [comment]
Please see v2.0.0-alpha1 in `master`. Does this work for you?

**Comment by sthristov** [comment]
Hi,

It works, thank you.

One thing that's worth mentioning for people following this issue is that if someone uses the default bash on a MacOS it won't work.
The bash dictionary declaration is supplied with bash version 4 and MacOS comes by default with version 3.

Cheers,
Stan

**Comment by Zordrak** [comment]
Yes, as per the CHANGELOG, 2.0.0-alpha1 depends on bash4. I have considered writing a bash3 associative array wrapper but it's a lot of effort.

**Comment by Zordrak** [comment]
This has also been fixed and now supports bash3
